app/src/main/java/com/m2049r/xmrwallet/service/exchange/api/ExchangeCallback.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2006 The Android Open Source Project
+ * Copyright (c) 2017 m2049r
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.m2049r.xmrwallet.service.exchange.api;
+
+public interface ExchangeCallback {
```

Comment:
there will be no other types - the exchange service is meant to deliver exchange rates and nothing else. even if it were to deliver another type it would be a subclass of ExchangeRate which has the minimal info for these services.